py3status/modules/sysdata.py

```diff
@@ -15,7 +15,8 @@
         (default '°C')
     thresholds: thresholds to use for color changes
         (default [(0, "good"), (40, "degraded"), (75, "bad")])
-    zone: thermal zone to use. If None try to guess CPU temperature
+    zone: Either a path in sysfs to CPU temperature sensor, or an lm_sensors thermal zone to use.
```

Comment:
It looks neat, but I'm only a bit worried that when people know they want to specify lm_sensors zone, trying to read a non-existent file and raising&catching exceptions every few seconds could be expensive operation, what do you think?

By the way, is specifying sysfs path superior and people better do this instead of lm_sensors zone anyway?

Reply:
I'm looking into this. I found [this](https://stackoverflow.com/questions/82831/how-do-i-check-whether-a-file-exists-without-exceptions) but it doesn't mention performance

Reply:
I'm of the opinion that the sysfs way is better. It pulls the same data that lm_sensors does. The catch is that you need to know where to look. :/

Reply:
In my experiment there's no perf difference, so I'm fine keeping everything as it is now.